src/components/date/date.spec.tsx

```diff
@@ -503,25 +631,6 @@ describe("Date", () => {
         expect(wrapper.update().find(DayPicker).exists()).toBe(false);
         wrapper.unmount();
       });
-
-      it("the 'DatePicker' should close if it is open", () => {
```

Comment:
question: has this test been removed intentionally?

Reply:
Figured there would have been but just wanted to make sure it was an accidental thing 👍🏻